### PR TITLE
Add Blade Submenu component

### PR DESCRIPTION
**Pull request by Jellyfrog**
A component used for creating submenus,

Feed it with an array like;
```php
'submenu' => [
    [
        ['name' => 'Basic', 'url' => ''],
    ],
    'Graphs' => [
        ['name' => 'Bits', 'url' => 'bits'],
        ['name' => 'Unicast Packets', 'url' => 'unicast'],
        ...
    ],
]
```
To output:
image

To discuss:
- [ ] Name
- [ ] Missing features

#### Please note

> Please read this information carefully. You can run `./lnms dev:check` to check your code before submitting.

- [x] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)
- [x] If my Pull Request does some changes/fixes/enhancements in the WebUI, I have inserted a screenshot of it.

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

### CODE REVIEW

**Comment by Jellyfrog** [comment]
See first usage in #12163

**Comment by murrant** [comment]
I dislike the style of these menus, but it will make them easier to change the style in the future :+1:

File: app/View/Components/Submenu.php [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+     *
+     * @return void
+     */
+    public function __construct($title, $menu, $deviceId, $currentTab, $selected)
+    {
+        $this->title = $title;
+        $this->menu = $menu;
+        $this->device_id = $deviceId;
+        $this->current_tab = $currentTab;
+        $this->selected = $selected;
```
Comment (murrant) [inline app/View/Components/Submenu.php:55]: Looks like you changed selected from a string to an array, but didn't fully convert the code.
Perhaps renaming it to $tabs or $items might be a good idea.

Comment (Jellyfrog) [reply app/View/Components/Submenu.php:55]: Hmm no, selected is a string with the current url To be able match currently selected tab.

**Review by murrant — APPROVED** [review]
Looks fine to me, but seems to be a device submenu component.

**Comment by murrant** [comment]
This pull request has been mentioned on **LibreNMS Community**. There might be relevant details there:

https://community.librenms.org/t/v1-69-release-changelog-october-2020/13837/1

**Pull request merged**